.github/workflows/release-publish.yml

```diff
@@ -216,7 +216,7 @@ jobs:
         LAST_TAG=$(git describe --abbrev=0 --tags --match=nessie-* ${GIT_TAG}^1)
         NUM_COMMITS=$(git log --format='format:%h' ${LAST_TAG}..HEAD^1 | wc -l)
 
-        git log --perl-regexp --author '^(?!.*dependabot(-preview)?|.*nessie-release-workflow).*$' --format='format:* %s' ${LAST_TAG}..${GIT_TAG} | grep -v '^\* \[release\] .*$' > ${DIR}/release-log
+        git log --perl-regexp --author '^(?!.*renovate|.*dependabot(-preview)?|.*nessie-release-workflow).*$' --format='format:* %s' ${LAST_TAG}..${GIT_TAG} | grep -v '^\* \[release\] .*$' > ${DIR}/release-log
```

Comment:
due to your recent PRs i think this adjustment needs to be done in more places now...

generally we should grep the repo for "dependabot" and make sure all remaining usage spots are necessary